examples/pytorch/ogb/directional_GSN/main.py

```diff
@@ -291,19 +207,8 @@ def forward(self, g, h, e, snorm_n):
         return self.MLP_layer(hg)
 
     def loss(self, scores, labels):
-        
-        loss = 0
-        loss_fn = torch.nn.BCEWithLogitsLoss()
-
-        for i in range(labels.shape[1]):
-            # ignore nan values
-            is_labeled = labels[:,i] == labels[:,i]
-
-            # otherwise loss will be 'nan'
-            if (len(labels[is_labeled, i]) == 0):
-                continue
-
-            loss += loss_fn(scores[is_labeled,i], labels[is_labeled,i].to('cuda'))
+        is_labeled = labels == labels
+        loss = torch.nn.BCEWithLogitsLoss()(scores[is_labeled], labels[is_labeled].float().to('cuda'))
```

Comment:
Does this work? Have you tried re-running the script?

Reply:
I have tested the model and the loss decreased normally.